app/uk/gov/hmrc/transitmovements/controllers/MovementsController.scala

```diff
@@ -71,7 +70,7 @@ class MovementsController @Inject() (
                   fileSource
                 )
                 .asPresentation
-              _ <- repo.updateMessages(DepartureId(movementId.value), message, messageData.mrn).asPresentation
+              _ <- repo.updateMessages(MovementId(movementId.value), message, messageData.mrn).asPresentation
```

Comment:
you can pass the movementId here, no need to create a new MovementId object and insert the value since the type is already as expected